metis-indexing/metis-indexing-client/src/main/java/eu/europeana/indexing/client/config/IndexingConfig.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2007-2013 The Europeana Foundation
+ *
+ *  Licenced under the EUPL, Version 1.1 (the "Licence") and subsequent versions as approved
+ *  by the European Commission;
+ *  You may not use this work except in compliance with the Licence.
+ *
+ *  You may obtain a copy of the Licence at:
+ *  http://joinup.ec.europa.eu/software/page/eupl
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under
+ *  the Licence is distributed on an "AS IS" basis, without warranties or conditions of
+ *  any kind, either express or implied.
+ *  See the Licence for the specific language governing permissions and limitations under
+ *  the Licence.
+ */
+package eu.europeana.indexing.client.config;
+
+import com.mongodb.MongoClient;
+import com.mongodb.ServerAddress;
+import eu.europeana.indexing.service.PublishingService;
+import eu.europeana.indexing.service.dao.FullBeanDao;
+import eu.europeana.indexing.service.mongo.MorphiaDatastoreProvider;
+import eu.europeana.metis.exception.IndexingException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import org.apache.commons.lang.StringUtils;
+import org.apache.solr.client.solrj.impl.CloudSolrServer;
+import org.apache.solr.client.solrj.impl.LBHttpSolrServer;
+
+public class IndexingConfig {

```

Comment:
This class uses only code from the service project. It would seem that there is not much reason to have two different projects here (client vs service): there is no service, just client. I would suggest merging these projects into one indexing project.

Reply:
Agreed. Fixed.